portal-impl/src/META-INF/portal-model-hints.xml

```diff
@@ -342,7 +342,7 @@
 		<field name="modifiedDate" type="Date" />
 		<field name="fileEntryTypeKey" type="String" />
 		<field localized="true" name="name" type="String">
-			<hint name="max-length">4000</hint>
+			<hint-collection name="CLOB" />
```

Comment:
I remember some regressions caused by this kind of data type changes in Oracle, like https://issues.liferay.com/browse/LPS-47261. Are we using this field in any DISTINCT clause?